php/github_asset.php

```diff
@@ -49,12 +49,13 @@ function check_url($url, $proto = false) {
   return array($username, $repository, $tag_or_branch, $folder, $world_or_proto);
 }
 
+function yaml_error($msg) {
+  return "YAML file error: $msg";
+}
+
```

Comment:
Why this change? It increases the scope of the yaml_error function, but that doesn't seem necessary...

Reply:
It is necessary because, when refreshing all the protos, the script calls the `github_check_yaml` function multiple times and it causes a ` Cannot redeclare yaml_error() ` error if this function is in the body of `github_check_yaml`